diary-volumes/volume19.xml

```diff
@@ -0,0 +1,209 @@
+            </div>
+            <div xml:id="EBA-1913-02-09" type="entry">
+                <dateline>
+                    <date when="1913-02-09">Sunday. Feb 9th.</date>
+                </dateline>
+                <p>It has been very warm - various visitors - among them the <persName ref="#Longyear_Mr #Longyear_Mrs">Longyears</persName> back from the South as far as <placeName ref="#Khartoum">Khartoum</placeName>. <persName ref="#Whymper_Charles">Mr. Whymper</persName> to breakfast this morning.</p>
+            </div>
+            <div xml:id="EBA-1913-02-16" type="entry">
+                <dateline>
+                    <date when="1913-02-16">Sunday. Feb. 16th.</date>
+                </dateline>    
+                <p>We dressed the ship on <date when="1913-02-12">Wednesday</date> - <persName ref="#Lincoln_Abraham">Lincoln</persName>'s birthday - the <pb n="110"/>usual routine - this week - we have had light rain on one day and cold stormy weather since. <persName ref="#Contardone_Contessa">Contessa</persName> here for luncheon.</p>
+            </div>
+            <div xml:id="EBA-1913-02-23" type="entry">
+                <dateline>
+                    <date when="1913-02-23">Sunday. Feb. 23rd.</date>
+                </dateline>
+                <p>Very warm. <persName ref="#Whymper_Charles">Mr. Whymper</persName> to breakfast - the <persName ref="#Pier_Mr #Pier_Mrs">Piers</persName> to lunch. <persName ref="#Aylener_Mrs">Mrs. Aylener</persName> and <persName ref="#Morehouse_Mrs">Morehouse</persName> <!--Mr. or Mrs.?-->to tea. We have had <persName ref="#Hardy_Mr #Hardy_Mrs">Mr. and Mrs. Hardy</persName> to tea - <persName ref="#Nelson_Aldrich">Senator Aldrich</persName> also. <persName ref="#Newberry_Mary">Mary N.</persName> and I have been over to the <name type="hotel" ref="#Winter_Palace">Winter Palace</name> for tea - we looked very smart on Washington's birthday with all our flags.</p>
+            </div>
+            <div xml:id="EBA-1913-03-02" type="entry">
+                <dateline>
+                    <date when="1913-03-02">Sunday. March 2nd.</date>
+                </dateline>
+                <p>On <date when="1913-02-24">Monday</date> <persName ref="#Newberry_Mary">Mary</persName> and I went to tea with the <persName ref="#Contardone_Contessa">Contessa</persName> in the gardens of the <name type="hotel" ref="#Luxor_Hotel">Luxor Hotel</name> - met <persName ref="#Baring_Lady">Lady Baring</persName>, <persName ref="#Murray_Miss">Miss Murray</persName>, and <persName ref="#Pappenheim_Contessa">Contessa Pappenheim</persName> and daughter <!--tag daughter?-->. Charming people all - yesterday we left for <placeName ref="#Aswan">Asw&#226;n</placeName>.</p>
```

Comment:
I don't think it'll be possible to find information on the daughter at this time